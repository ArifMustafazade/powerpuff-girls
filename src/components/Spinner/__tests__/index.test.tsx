import React from 'react';
import { ShallowWrapper, shallow } from 'enzyme';

// components
import { Spinner } from '../';

/** tests */
describe('<Spinner />', () => {
    let wrapper: ShallowWrapper;
    beforeEach(() => {
        wrapper = shallow(<Spinner />);
    });

    test('should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
