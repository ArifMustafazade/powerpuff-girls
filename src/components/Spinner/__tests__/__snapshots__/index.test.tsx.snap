// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner /> should match snapshot 1`] = `
<div
  className="spinner"
>
  <img
    alt="Spinner"
    src="animated_spinner.webp"
  />
</div>
`;
